action/index.js

```diff
@@ -72,8 +75,8 @@ async function main() {
   core.debug(`Discovering assets in "${cwd}"`);
   const assets = new Map();
   const entries = await walk(cwd, cwd, assets, {
-    include: undefined,
-    exclude: undefined,
+    include: include ? include.split(",").map(pattern => normalize(pattern)): undefined,
```

Comment:
Newline separated strings is the defacto-standard workaround to express arrays in GH action parameters. `core.getMultilineInput()` handles splitting and trimming for you. 

It's preferrable as there might be array parameters we want to add in the future that might not play as well with the comma-separated approach.